pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -235,8 +235,8 @@ func (cb *ClusterBuilder) buildDefaultCluster(name string, discoveryType cluster
 }
 
 func (cb *ClusterBuilder) buildInboundClusterForPortOrUDS(instance *model.ServiceInstance, bind string) *cluster.Cluster {
-	clusterName := model.BuildSubsetKey(model.TrafficDirectionInbound, instance.ServicePort.Name,
-		instance.Service.Hostname, instance.ServicePort.Port)
+	clusterName := model.BuildSubsetKey(model.TrafficDirectionInbound, "",
+		"inbound", instance.ServicePort.Port)
```

Comment:
Why is inbound is repeated twice?

Reply:
I can probably make it empty. The idea is to keep the `.|.|.|.` format since a lot of tools rely on it. But we can probably do `inbound|port||` instead of  `inbound|port||inbound`?